components/configuration-mgt/org.wso2.carbon.identity.configuration.mgt.endpoint/src/main/resources/org.wso2.carbon.identity.configuration.management.yaml

```diff
@@ -505,17 +505,17 @@ paths:
       x-wso2-response: |
         {"id": "42c8d3f1-48f9-40e7-98e2-2b6376731fc0","key": "from","value": "abc.com"}
       summary: |
-        Add or Replace an attribute
+        Create or Replace attribute
       # put parameters:
       parameters:
         - name: resource-name
           in: path
-          description:  This represents the name of the attribute to be added or replaced.
+          description:  This represents the name of the attribute that is to be added or replaced.
           required: true
           type: 'string'
         - name: resource-type
           in: path
-          description:  This represents the type of the attribute to be added or replaced and can either be the name or id.
+          description:  This represents the type of attribute that is to be added or replaced and can either be the resource-type name or ID.
```

Comment:
This value can only be the name, not the id. Shall we change it accordingly?

Reply:
Fixed with [c2af0a2](https://github.com/wso2/carbon-identity-framework/commit/c2af0a29f9313baf5105a4898a9e26baceae00fa)